Build errors

I'm getting the following errors when running "mvn clean package":

```
[INFO] --- xtend-maven-plugin:2.7.3:compile (default) @ antlr4ide.core ---
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
9: org.abego.treelayout.util.DefaultTreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
11: org.abego.treelayout.TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
106: TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
164: TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
261: TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
221: DefaultTreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
256: TreeForTreeLayout cannot be resolved to a type.
[ERROR] 
ERROR:  ParseTreeGenerator.xtend - 
241: DefaultTreeForTreeLayout cannot be resolved.

[INFO] Reactor Summary:
[INFO] 
[INFO] antlr4ide ......................................... SUCCESS [  0.037 s]
[INFO] ANTLR v4 IDE Runtime .............................. SUCCESS [  1.636 s]
[INFO] ANTLR v4 IDE Core ................................. FAILURE [ 25.559 s]
[INFO] ANTLR v4 IDE UI ................................... SKIPPED
[INFO] ANTLR v4 IDE Tests ................................ SKIPPED
[INFO] ANTLR v4 IDE Application .......................... SKIPPED
[INFO] ANTLR v4 IDE SDK .................................. SKIPPED
[INFO] ANTLR 4 IDE - Update Site ......................... SKIPPED
```

Please advise.

Thanks!
-Larry

--- a/README.md
+++ b/README.md
@@ -146,13 +146,17 @@ Building ANLTR 4 IDE
 =========
 
 1. Build with Maven 3.x
+
   1. Fork and clone the repository from github
   2. Download and install [Maven 3.x](http://maven.apache.org/)
   3. Open a shell console and type: ```cd antlr4ide```
   4. Build the project with: ```mvn clean package```
   5. It takes a while to download and configure Eclipse dependencies, so be patient
   6. Wait for a: ```BUILD SUCCESS``` message
+  7. When building for the very first time there might be some missing dependencies, so please retry ```mvn clean package``` 
+  
 2. Setup Eclipse
+
   1. Open Eclipse Luna (4.4)
   2. Install Xtext 2.7.3
   3. Copy and paste this url: http://download.eclipse.org/modeling/tmf/xtext/updates/composite/releases/ in the **Work with** text field

